pkg/dashboard/resource/function.go

```diff
@@ -777,7 +836,58 @@ func (fr *functionResource) populateGetFunctionReplicaLogsStreamOptions(request
 	}
 
 	return getFunctionReplicaLogsStreamOptions, nil
+}
+
+func (fr *functionResource) populateProxyFunctionLogsOptions(request *http.Request, functionName string) (*platform.ProxyFunctionLogsOptions, error) {
+	proxyFunctionLogsOptions := platform.NewProxyFunctionLogsOptions(functionName)
+	proxyFunctionLogsOptions.TimeFilter = fr.getURLParamTimeFilterOrDefault("timeFilter", request, &platform.TimeFilter{
+		Sort: "desc",
+	})
+	proxyFunctionLogsOptions.Substring = fr.GetURLParamStringOrDefault(request, "substring", "")
+	proxyFunctionLogsOptions.Regexp = fr.GetURLParamStringOrDefault(request, "regexp", "")
+	proxyFunctionLogsOptions.ReplicaNames = fr.GetURLParamStringValues("replicaNames", request)
+	proxyFunctionLogsOptions.LogLevels = fr.GetURLParamStringValues("logLevels", request)
+	proxyFunctionLogsOptions.Size = fr.GetURLParamInt64OrDefault(request, "size", 100)
+	proxyFunctionLogsOptions.SearchAfter = fr.GetURLParamStringValues("searchAfter", request)
+	proxyFunctionLogsOptions.Source = platform.ProxyLogsSource(fr.GetURLParamStringOrDefault(
+		request,
+		"source",
+		string(platform.ProxyLogsSourceES),
```

Comment:
Why is this the default? what if it is not configured at all?
Default should be conditional on the ES configuration existence.

Reply:
@TomerShor added a mechanism to resolve a default.